### PR TITLE
Add packages section to README

### PR DESCRIPTION
**Pull request by jloeser**
Referring to https://github.com/Foxboron/sbctl/issues/167#issuecomment-1477121243

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,16 @@ testing towards [tianocore](https://www.tianocore.org/) utilizing
 * binutils (using `objcopy`)
 * Go >= 1.16
 
+## Available sbctl Packages
+
+List of existing sbctl packages can be found
+[here](https://repology.org/project/sbctl/versions).
```
Comment (dawidpotocki) [inline README.md:40]: Don't use "here" for hyperlinks:
https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#use_clear_link_wording

```suggestion
[on Repology](https://repology.org/project/sbctl/versions).
```

File: README.md [hunk 2]
```diff
@@ -34,6 +34,16 @@ testing towards [tianocore](https://www.tianocore.org/) utilizing
...
+
+In addition, sbctl is also available for [openSUSE &
+Ubuntu](https://software.opensuse.org/package/sbctl?search_term=sbctl). Follow
+the `Expert Download` links to find installation instructions according to your
+operating system.
```
Comment (dawidpotocki) [inline README.md:45]: Should probably mention that Ubuntu is unofficial and openSUSE is experimental.

**Review by dawidpotocki — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -34,6 +34,16 @@ testing towards [tianocore](https://www.tianocore.org/) utilizing
 * binutils (using `objcopy`)
 * Go >= 1.16
 
+## Available sbctl Packages
```
Comment (dawidpotocki) [inline README.md:37]:
```suggestion
## Available sbctl packages
```

File: README.md (hunk 1 above)
Comment (dawidpotocki) [inline README.md:40]: Don't use "here" for hyperlinks:
https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Creating_hyperlinks#use_clear_link_wording

```suggestion
You can find a list of
[sbctl packages on Repology](https://repology.org/project/sbctl/versions).
```

File: README.md (hunk 2 above)
Comment (jloeser) [reply README.md:45]: What is the difference between unofficial and experimental?

File: README.md (hunk 3 above)
Comment (jloeser) [reply README.md:37]: I am used to https://en.wikipedia.org/wiki/Title_case.

As this is not used in the rest of the README I am fine with this.

File: README.md (hunk 2 above)
Comment (dawidpotocki) [reply README.md:45]: Well, to me it looks like the openSUSE package is kind of in a "testing" repository, while Ubuntu package is maintained by the community.

**Comment by Foxboron** [comment]
This works as a first iteration. I wanted to include a `go install` command thing and some `pacman -S` install instructions as well. It's unclear to me how that repository is actually used on suse and debian but works :) Thanks!

**Review by Foxboron — APPROVED** [review]

**Review by dawidpotocki — APPROVED** [review]
I'm not a maintainer, but if you want it from me… I approve I guess

**Comment by Foxboron** [comment]
I haven't forgottent his :)

Havent had time to take a round of merging approved PRs the past couple of weeks because I'm moving and some work travel.

**Comment by Foxboron** [comment]
Thanks!

**Pull request merged**